x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/create_rules_route.ts

```diff
@@ -78,9 +79,15 @@ export const createRulesRoute = (
 
         // This will create the endpoint list if it does not exist yet
         await ctx.lists?.getExceptionListClient().createEndpointList();
+        const exceptionLists = checkDefaultRuleExceptionListReferences({
+          exceptionLists: request.body.exceptions_list,
+        });
         const createdRule = await createRules({
           rulesClient,
-          params: request.body,
+          params: {
+            ...request.body,
+            exceptions_list: exceptionLists,
```

Comment:
What's the functional effect of using `exceptionLists` here? I think we might not need to change this call.

Reply:
Yea, you're right. Updated!